src/ucp/core/ucp_context.h

```diff
@@ -141,6 +143,8 @@ struct ucp_config {
     char                                   *selection_cmp;
     /** Configuration saved directly in the context */
     ucp_context_config_t                   ctx;
+    /** Save effective ucx configurations not listed in ucp_config_table **/
+    ucs_list_link_t                        list;
```

Comment:
maybe rename to "extra_keys_list" or "cached_key_list" ?

Reply:
Done, renamed to ```cached_key_list```